<launch>
  <arg name="ipfs_http_provider" default="http://127.0.0.1:5001" />
  <arg name="server_address" default="https://0.0.0.0:4321" />
  <arg name="certfile" default="" /> <!-- your "domain.cer" file path -->
  <arg name="keyfile" default="" /> <!-- your "domain.key" file path -->

  <node name="objective_composer" pkg="de_objective_composer" type="objective_composer_node" output="screen">
    <param name="ipfs_http_provider" value="$(arg ipfs_http_provider)" />
    <param name="server_address" value="$(arg server_address)" />
    <param name="certfile" value="$(arg certfile)" /> <!-- SSL certificate for webserver -->
    <param name="keyfile" value="$(arg keyfile)" />
  </node>
</launch>
